index.html

```diff
@@ -5155,6 +5155,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
 
 </section>
 
+<section id="presentation-media-type">
+  <h2>The <code>application/presentation+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/presentation+ld+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Presentations format.
```

Comment:
Why use `verifiable presentations ` in the description of a media type for a `presentation`?

```suggestion
    conforming to the Presentations format.
```

Reply:
technical `verifiable presentations` NEVER required a `proof`.... I prefer to keep this sentence as is, to align with the way we registered `credential+ld+json`.

Reply:
however, we we feel very strongly about it, am willing to merge if we create an issue to make both registrations consistent.

Reply:
Suggest that we keep it the same as `application/credential+ld+json` for now. We should raise an issue to clarify the language to be consistent across both media types. I've raised issue #1052 to track @Sakurann's concern... which should allow us to merge this PR as long as @Sakurann is ok w/ that approach.

Reply:
I'm fine with the format/media-type(s) always being labeled as Verifiable, as long as it's made clear everywhere that the Verifiability is not determined by conformance with the Media Type, but by the presence and provability of the optional proof(s), the lack of which deliver an UNverifiable Presentation/Credential.